jujugui/static/gui/src/app/components/entity-files/_entity-files.scss

```diff
@@ -0,0 +1,15 @@
+.entity-files {
+    ul {
```

Comment:
All of which is to say: I'm all for BEM. I just want to be pragmatic about it rather than dogmatic.